### PR TITLE
K-70-incorrect-GCP-registry-setting

### PR DESCRIPTION
**Pull request by WiktorJ**
Changed from gcloud config to tf variable

### CODE REVIEW

File: scripts/gcp/build_push_tag.sh [hunk 1]
```diff
@@ -41,7 +45,6 @@ DOCKER_FILE=${DOCKER_FILE:-${DOCKER_CONTEXT}/Dockerfile}
 echo "==========================="
 echo "| Configure GCP region... |"
 echo "==========================="
-REGION=$(gcloud config get-value compute/region)
 if [[ -z "$REGION" ]]
     then
```
[the remark this answers is not in the document]
Comment (WiktorJ) [reply scripts/gcp/build_push_tag.sh]: Yea, good point, we should remove it.

File: infrastructure/gcp/modules/repository/main.tf [hunk 2]
```diff
@@ -46,7 +46,7 @@ resource "null_resource" "build_push_tag" {
   }
 
   provisioner "local-exec" {
-    command = "sh ${var.scripts_dir}/gcp/build_push_tag.sh -c=${element(var.docker_contexts, count.index)} -f=${element(var.docker_files, count.index)} -i=${element(var.image_names, count.index)} -t=${element(var.image_tags, count.index)} -p=${var.project_name}"
+    command = "sh ${var.scripts_dir}/gcp/build_push_tag.sh -c=${element(var.docker_contexts, count.index)} -f=${element(var.docker_files, count.index)} -i=${element(var.image_names, count.index)} -t=${element(var.image_tags, count.index)} -p=${var.project_name} -r=${var.region}"
```
[the remark this answers is not in the document]
Comment (WiktorJ) [reply infrastructure/gcp/modules/repository/main.tf]: The problem is that there is no such region in gcloud config. The only thing user can set is the default location in the form of `eu-west-1` for example. So the only thing we can do is to infer this region from the more specific location. 
But even if we do that, we still have to give users the option to deploy in a different region then default one, so this ENV has to be present anyway.

[the remark this answers is not in the document]
Comment (WiktorJ) [reply infrastructure/gcp/modules/repository/main.tf]: For now our config is totally independent from gcloud config (also when it comes to cluster and network locations), which I think is good, as we are less coupled with the gcloud tool.

[the remark this answers is not in the document]
Comment (jfri3d) [reply infrastructure/gcp/modules/repository/main.tf]: Yeah - I see that the regions don't align perfectly. Do you suggest us to expose the region as an ENV then? Would need to be in the docs then...

[the remark this answers is not in the document]
Comment (WiktorJ) [reply infrastructure/gcp/modules/repository/main.tf]: Yea, I think it's better to expose it, I'll update the docs for that.

**Pull request merged**